src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -225,4 +229,4 @@ class Comment extends React.Component<Props, State> {
     }
 }
 
-export default Comment;
+export default injectIntl(Comment);
```

Comment:
It may be safer to move the entire menu to its own component and wrap that with `injectIntl`, since this seems to have had some consequences for the parent component.

Reply:
did not understand this comment
may you elaborate, please

Reply:
do you mean to create something like CommentMenu component then import it and use it here?

Reply:
Yep, that was my initial idea for a solution if `injectIntl` is interfering with refs/callbacks as the snapshots imply.

Reply:
you are completely right but we are running out of time since first deliverable to webAIM have to be send for tomorrow.
so we are going to close this issue for the time being and resume after the receive feedback from webAIM.